themes/bootstrap3/templates/RecordTab/similaritemscarousel.phtml

```diff
@@ -1,6 +1,6 @@
 <h2><?=$this->transEsc('Similar Items')?></h2>
 <?php $similarRecords = $this->tab->getResults(); ?>
-<?php if (!empty($similarRecords)): ?>
+<?php if (!$similarRecords->isEmpty()): ?>
```

Comment:
Would a simpler solution be to change `empty` to `count` in all of these places? Since the record collection is countable, I think it would have the desired effect, and then there would be no need to add a custom empty-checking method.

What do you think?

Reply:
We could do either `$similarRecords->count()` or `count($similarRecords)`; for general collections, of course, testing emptiness may be cheaper than counting the elements, but I don't think this is an issue here. Will update the PR accordingly.